src/vl53l5cx_class.h

```diff
@@ -60,10 +60,10 @@ class VL53L5CX {
      * @param[in] lpn_pin pin to be used as component LPn
      * @param[in] i2c_rst_pin pin to be used as component I2C_RST
      */
-    VL53L5CX(TwoWire *i2c, int lpn_pin, int i2c_rst_pin = -1)
+    VL53L5CX(TwoWire *i2c, int lpn_pin, int i2c_rst_pin = -1, uint16_t default_i2c_address = VL53L5CX_DEFAULT_I2C_ADDRESS)
```

Comment:
I do not agree to pass the I2C address in the constructor because the first command to change the I2C address must be done with the default I2C address of the device, otherwise it does not work.